src/i18n/fi.json

```diff
@@ -253,21 +253,26 @@
       "sendSuccess": "Lähetys onnistui",
       "patchSuccess": "Muokkaus onnistui",
       "deleteSuccess": "Poisto onnistui",
+      "401": "Virhe kirjautumisessa",
       "404": "Virhe, resurssia ei löytynyt",
-      "400": "Vääränlainen pyyntö",
+      "403": "Kielletty toiminto",
+      "400": "Virheellinen pyyntö",
       "500": "Palvelinvirhe",
       "projectNotFound": "Hanketta ei löytnyt",
       "hashtagPatchSuccess": "Tunniste päivitetty",
       "patchError": "Päivitys epäonnistui",
       "saveError": "Tallennus epäonnistui",
       "hashtagPostSuccess": "Tunniste tallennettu",
-      "postError": "Luonti epäonnistui"
+      "postError": "Luonti epäonnistui",
+      "accessDenied": "Pääsy evätty"
     },
     "message": {
       "formSaveSuccess": "Lomake tallennettu onnistuneesti.",
       "noteDeleteSuccess": "Muistiinpano poistettu onnistuneesti",
       "notePatchSuccess": "Muistiinpano muokattu onnistuneesti",
       "Request failed with status code 400": "Pyyntö epäonnistui virhekoodilla 400",
+      "Request failed with status code 401": "Pyyntö epäonnistui virhekoodilla 401",
+      "Request failed with status code 403": "Pyyntö epäonnistui virhekoodilla 403",
```

Comment:
Is it okay that there are spaces between the words?

Reply:
I agree that it's not the best way to name these, but these are the error messages that thunkAPI is creating when posting an error and for some reason they can't be modified directly. Another way to do this would be to have a converter for the messages that changes it to some better variable name that is then used for getting the translation.

Reply:
Okay I see, I think that we can keep it this way now and maybe come back to it later